input/resources/au-medication.xml

```diff
@@ -32,61 +32,56 @@
     <element id="Medication.code">
       <path value="Medication.code"/>
       <short value="Coding for the medicine"/>
-      <comment
-        value="Australian coding slices are typical medicine/product concept codes.&#xD;&#xA;&#xD;&#xA;A code (or set of codes) that specify this medication, or a textual description if no code is available. Usage note: This could be a standard medication code such as a code from RxNorm, SNOMED CT, IDMP etc. It could also be a national or local formulary code, optionally with translations to other code systems."
-      />
-    </element>
-    <element id="Medication.code.coding">
-      <path value="Medication.code.coding"/>
-      <slicing>
-        <discriminator>
-          <type value="value"/>
-          <path value="system"/>
-        </discriminator>
-        <rules value="open"/>
-      </slicing>
-    </element>
-    <element id="Medication.code.coding:pbs">
-      <path value="Medication.code.coding"/>
-      <sliceName value="pbs"/>
-      <short value="PBS Item Code"/>
-      <comment
-        value="PBS code from http://pbs.gov.au/code/item. Use of PBS as a code to refer to a type of medication, this excludes implication of context based on the PBS code. Where context is to be implied or stated PBS code needs to be associated with recording a prescription (MedicationRequest) or dispense record (MedicationDispense)."/>
+      <comment value="Australian coding are typical medicine/product concept codes.&#xD;&#xA;&#xD;&#xA;A code (or set of codes) that specify this medication, or a textual description if no code is available. Usage note: This could be a standard medication code such as a code from RxNorm, SNOMED CT, IDMP etc. It could also be a national or local formulary code, optionally with translations to other code systems."/>
       <binding>
-        <strength value="preferred"/>
-        <valueSet value="http://terminology.hl7.org.au/ValueSet/pbs-item"/>
-      </binding>
-    </element>
-    <element id="Medication.code.coding:pbs.system">
-      <path value="Medication.code.coding.system"/>
-      <min value="1"/>
-      <fixedUri value="http://pbs.gov.au/code/item"/>
-    </element>
-    <element id="Medication.code.coding:gtin">
-      <path value="Medication.code.coding"/>
-      <sliceName value="gtin"/>
-      <short value="Medication Package Global Trade Item Number (GTIN)"/>
-      <comment value="GTIN value from http://www.gs1.org/gtin."/>
-      <binding>
-        <strength value="preferred"/>
-        <valueSet value="http://terminology.hl7.org/ValueSet/v3-GTIN"/>
+        <extension url="http://hl7.org/fhir/tools/StructureDefinition/additional-binding">
+          <extension url="purpose">
+            <valueCode value="preferred"/>
+          </extension>
+          <extension url="valueSet">
+            <valueCanonical value="https://healthterminologies.gov.au/fhir/ValueSet/australian-medication-1"/>
```

Comment:
This opens to https://tx.ontoserver.csiro.au/fhir/ValueSet/australian-medication-1

Reply:
@JayMurdoch - that's nice. What's the review comment, i.e. what has to change and why?

Reply:
I now understand this is a known issue, and no change is required. Thanks.

Reply:
This comment is now resolved.